Intro to open source with GitHub/README.md

```diff
@@ -0,0 +1,22 @@
+# How to work on opensource contribution with Git
+
+## Making a contribution
+1. Fork the repo which you want to work on (eg. https://github.com/take2rohit/sih_number_plate is original repo where you want to contribute) 
```

Comment:
eg. [this repo](link) is the original repository you want to contribute to

Reply:
I dont think its a good idea.